packages/preset-rules/__tests__/rules.test.ts

```diff
@@ -17,16 +17,11 @@ describe('preset-rules', () => {
     // or due to their experimental nature
     const excludedRules = [
         /* cSpell:disable */
-        'aria-dpub-role-fallback',
```

Comment:
@cordeliadillon Changes in this file to remove rules that have been removed from axe v4 along with updating help urls of a11y results in test snapshots were the primary changes to upgrade to axe v4. Can you please review.

Reply:
The removed rules match https://github.com/dequelabs/axe-core/releases/tag/v4.0.0
```
The following rules were deprecated in axe-core 3.x, and are removed in 4.0:

aria-dpub-role-fallback
checkboxgroup
layout-table
radiogroup
video-description
```